lib/view_component/collection.rb

```diff
@@ -35,5 +36,18 @@ def component_options(item, iterator)
 
       @options.merge(item_options)
     end
+
+    def ensure_initializer_accepts_iterator
```

Comment:
Rails convention might be to put a `! on the end of this method given that it will raise